js/components/graph/Graph.js

```diff
@@ -159,6 +162,20 @@ export default class Graph extends React.Component {
             edgesList.push(entry);
           }
         });
+
+        // An object mapping a node to its parents/childs improves efficiency in NodeGroup/BoolGroup
```

Comment:
Remove this comment. The term "improves" implies a comparison or change that won't make sense once the old code is deleted. This is a situation where the "why" is best described in the pull request description.